sw.js

```diff
@@ -106,12 +83,7 @@ self.onactivate = event => {
 };
 
 router.registerRoutes({
-  routes: [
-    assetRoute,
-    externalRoute,
-    cdnAssetRoute,
-    navRoute
-  ]
+  routes: [assetRoute, externalRoute, cdnAssetRoute, navRoute]
```

Comment:
The `externalRoute` reference should be deleted since it was removed. 😬 
```
Uncaught ReferenceError: externalRoute is not defined
(anonymous) @ sw.js:86
```

Reply:
I'm not sure if it's related or not, but I also see this in the DevTools console:
```
Uncaught (in promise) TypeError: Failed to register a ServiceWorker: ServiceWorker script evaluation failed
```